src/test/java/com/thealgorithms/strings/ReverseWordsInStringTest.java

```diff
@@ -0,0 +1,23 @@
+package com.thealgorithms.strings;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ReverseWordsInStringTest {
+
+    @Test
+    public void testCorrectReverseWordsInTheString() {
+        assertEquals("blue is Sky", ReverseWordsInString.reverseWordsInString("Sky is blue"));
+    }
+
+    @Test
+    public void testCorrectReverseWordsInTheStringWithWhiteSpace() {
+        assertEquals("blue is Sky", ReverseWordsInString.reverseWordsInString("Sky \n is \n \n blue"));
+    }
+
+    @Test
+    public void testReverseWordsInStringForEmpty() {
+        assertEquals("", ReverseWordsInString.reverseWordsInString(""));
+    }
```

Comment:
The _logic_ in all of the tests is exactly the same. What do you think about expressing them as _parameterized tests_ as in [`FindMaxRecursionTest.java`](https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/maths/FindMaxRecursionTest.java)?

Reply:
done in 6bdd7d913c7b539a72290bfe7dc36df99f59e343